packages/ui/src/validators/hooks/useValidatorsList.tsx

```diff
@@ -0,0 +1,165 @@
+      controllerAccount: 'j4Rh1cHtZFAQYGh7Y8RZwoXbkAPtZN46FmuYpKNiR3P2Dc2oz',
+      boundAccounts: [],
+      inviteCount: 0,
+      roles: [],
+      isVerified: true,
+      isFoundingMember: true,
+      isCouncilMember: true,
+      createdAt: '2023/08/01',
+      entry: {
+        type: 'genesis',
+      },
+      invitees: [],
+      externalResources: [
+        { source: 'TWITTER', value: 'Alice_twitter' },
+        { source: 'DISCORD', value: 'Alice_discord' },
+        { source: 'TELEGRAM', value: 'Alice_telegram' },
+      ],
+    } as MemberWithDetails,
+  },
+  {
+    address: 'j4RjraznxDKae1aGL2L2xzXPSf8qCjFbjuw9sPWkoiy1UqWCa',
+    membership: {
+      id: '1',
+      handle: 'bob',
+      rootAccount: 'j4RjraznxDKae1aGL2L2xzXPSf8qCjFbjuw9sPWkoiy1UqWCa',
+      controllerAccount: 'j4RjraznxDKae1aGL2L2xzXPSf8qCjFbjuw9sPWkoiy1UqWCa',
+      boundAccounts: [],
+      inviteCount: 0,
+      roles: [],
+      isVerified: true,
+      isFoundingMember: false,
+      isCouncilMember: false,
+      createdAt: '2023/08/02',
+      entry: {
+        type: 'genesis',
+      },
+      invitees: [],
+      externalResources: [
+        { source: 'TWITTER', value: 'Bob_twitter' },
+        { source: 'DISCORD', value: 'Bob_discord' },
+      ],
+    } as MemberWithDetails,
+  },
+  { address: 'j4RuqkJ2Xqf3NTVRYBUqgbatKVZ31mbK59fWnq4ZzfZvhbhbN', membership: undefined },
+  { address: 'j4RxTMa1QVucodYPfQGA2JrHxZP944dfJ8qdDDYKU4QbJCWNP', membership: undefined },
+]
+
+const getMember = (address: Address) =>
+  validatorsWithMembership.find((validator) => validator.address === address)?.membership
+
+export const useValidatorsList = () => {
+  const { api } = useApi()
+  const [search, setSearch] = useState('')
+  const [isVerified, setIsVerified] = useState<Verification>(null)
+  const [isActive, setIsActive] = useState<State>(null)
+  const [visibleValidators, setVisibleValidators] = useState<Validator[]>([])
+
+  const getValidatorInfo = (address: string, api: Api): Observable<Validator> => {
+    const activeValidators$ = api.query.session.validators()
+    const stakingInfo$ = api.query.staking
+      .activeEra()
+      .pipe(switchMap((activeEra) => api.query.staking.erasStakers(activeEra.unwrap().index, address)))
+    const rewardHistory$ = api.derive.staking.stakerRewards(address)
+    const validatorInfo$ = api.query.staking.validators(address)
+    return combineLatest([activeValidators$, stakingInfo$, rewardHistory$, validatorInfo$]).pipe(
+      map(([activeValidators, stakingInfo, rewardHistory, validatorInfo]) => {
+        const encodedAddress = encodeAddress(address)
+        const apr =
+          rewardHistory.length &&
+          stakingInfo.total.toNumber() &&
+          last(rewardHistory)
+            .eraReward.toBn()
+            .muln(ERAS_PER_YEAR)
+            .mul(validatorInfo.commission.toBn())
+            .div(stakingInfo.total.toBn())
+            .divn(10 ** 7)
```

Comment:
Please keep this comment because it's not obvious for me:
```suggestion
            .divn(10 ** 7) // Convert from Perbill to Percent
```

Reply:
ok